helm/chart/router/values.yaml

```diff
@@ -77,6 +77,15 @@ containerPorts:
   # -- For exposing the health check endpoint
   health: 8088
 
+# An array of extra containers
```

Comment:
We use [helm-docs](https://github.com/norwoodj/helm-docs) to generate https://github.com/apollographql/router/tree/dev/helm/chart/router#readme. It takes comments in this format:

```
# -- Set to existing PriorityClass name to control pod preemption by the scheduler
```
to populate the **Description** field in the table. I really should put a comment about this at the top of the values.yaml file at some point...

Anyway: NIT:
```suggestion
# -- An array of extra containers to include in the router pod
```